scripts/src/main/resources/scripts/command/lazydocker

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+# Call lazydocker with specified arguments.
+function doRun() {
+  doSetup silent
+  cd ${LAZYDOCKER_HOME}
+  if doIsWindows
+  then
+    read -r -p "inside if condition of doRun"
+    powershell.exe -command 'start powershell {.\lazydocker.exe}'
+  else
+    ./lazydocker 
+  fi
```

Comment:
Seems that lazydocker has some issues to run on windows seamless in any shell.
Actually my vision would be that you could just setup lazydocker and then do not need `devon` command and just run `lazydocker` in any shell on any os.
Is that vision out of reach?
Can you share what errors or problems arise when you try this?

Reply:
This comments issue will be addressed in #842 as recently discussed and the comment can therefore be closed.